File: easy/020. Avik and his Girlfriend's Necklace/java/Main.java
import java.util.Scanner;

class Main {
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int n = input.nextInt();
		PartitionSort numbers = new PartitionSort(n);

		for (int i = 0; i < n; i++) {
			numbers.Insert(input.nextInt());
		}

		numbers.Sort();
		numbers.Show();
	}
}

File: easy/020. Avik and his Girlfriend's Necklace/c/main.c
#include <stdio.h>
#include "partition.h"

int main()
{
	int n = 0;
	scanf("%d", &n);

	int numbers[n];

	for (int i = 0; i < n; i++) {
		scanf("%d", &numbers[i]);
	}

	SortArray(numbers, 0, n-1);
	Show(numbers, &n);

	return 0;
}